app/models/ons_constituency.rb

```diff
@@ -6,4 +6,8 @@ class OnsConstituency < ApplicationRecord
   has_many :recommendations,
            primary_key: "ons_id",
            foreign_key: "constituency_ons_id"
+
+  def parties_by_marginal_score
```

Comment:
Where is this used?  And is this ordered by score descending or ascending?

Reply:
You're right ... not used ... I should apply YAGNI and delete, particularly as it's trivial to recreate

Reply:
I see your comment @aspiers so I will leave alone
> Adam Spiers 10:14 AM
> @baob  That method might be useful when debugging on console
> so I suggest keeping